packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -361,7 +358,13 @@ export default class DropdownCore extends React.Component<
         } else {
             this.focusedIndex -= 1;
         }
-        this.focusCurrentItem();
+
+        // force react-window to scroll on the correct position
+        this.listRef.current &&
```

Comment:
Minor nit, could you use an explicit if statement for this check?

```
if (this.listRef.current) {
    this.listRef.current.scrollToItem(this.focusedIndex);
}
```

I find these to be easier to understand (especially when we're not returning a value)